src/tests/ftest/util/soak_utils.py

```diff
@@ -480,6 +477,55 @@ def launch_snapshot(self, pool, name):
         "<<<PASS %s: %s completed at %s>>>\n", self.loop, name, time.ctime())
 
 
+def launch_vmd_identify_check(self, name, results, args):
+    """Run dmg cmds to blink/check VMD leds.
+
+    Args:
+        self (obj): soak obj
+        name (str): name of dmg subcommand
+        results (queue): multiprocessing queue
+        args (queue): multiprocessing queue
+    """
+    status = True
+    self.dmg_command.json.value = True
+    failing_vmd = []
+    result = self.dmg_command.storage_query_list_devices()
+
+    data = json.loads(result.stdout_text)
+    resp = data['response']
+    uuids = []
+    for value in list(resp['host_storage_map'].values()):
+        if value['storage']['smd_info']['devices']:
+            for device in value['storage']['smd_info']['devices']:
+                uuids.append(device['uuid'])
+
+    self.log.info("VMD device UUIDs: %s", uuids)
+
+    for uuid in uuids:
+        # Blink led
+        self.dmg_command.storage_led_identify(ids=uuid)
+        time.sleep(5)
+        # check if led is blinking
+        result = self.dmg_command.storage_led_check(ids=uuid)
+        # determine if leds are blinking as expected
+        for value in list(result['response']['host_storage_map'].values()):
+            if value['storage']['smd_info']['devices']:
+                for device in value['storage']['smd_info']['devices']:
+                    if device['led_state'] != "QUICK_BLINK":
```

Comment:
Expecting the led_state = "QUICK_BLINK" to pass might not be appropriate.
To my understanding, normal led state is "OFF", 
"QUICK_BLINK" seems a warning of the led device.

Reply:
@rpadma2 @dinghwah When we met last week I thought I was looking for QUICK_BLINK led state after doing the identify cmd.    the dmg storage led identify caused the LED to blink and then the dmg storage led check was looking for the QUICK_BLINK to verify that the identify was working?       I did try this on aurora and that did seem to be the case.
Please let me know if this is correct...

Reply:
I talked to Tom today and and get information of storage dev_state should be "NORMAL" and led_state be "OFF"  on normal working condition.  I tested the led identify --reset will reset all the storage devices led_state to "OFF".

Reply:
@mjean308  : Maybe, you should perform the --reset before soak completes (or) in teardown state. Otherwise, you will be leaving the LEDs in QUICK_BLINK state permanently.